leptos-rs/leptos
# fix(examples): unable to parse makefile (#1440)

Fixes #1440

### Changes

- Refresh the list of workspace members (via `gen-members`)

### Verification

1. Run `cargo make check-examples` from the `root` directory
2. See any member being compiled
